docs-chef-io/content/server/server_backup_restore.md

```diff
@@ -116,12 +116,29 @@ Restoring Chef Backend for a Chef Infra Server cluster has two steps:
     chef-backend-ctl join-cluster --accept-license --yes --quiet 198.51.100.0 --publish_address 203.0.113.0 -s /tmp/chef-backend-secrets.json
     ```
 
+    In case of corrupted node join cluster may fail, follow below steps to cleanse the node
```

Comment:
Are people at risk of losing data with `cleanse`? 
I'm looking at the description here https://docs.chef.io/server/ctl_chef_backend/#cleanse and looks like there's a risk there. 
Is chef-backend.rb file backed up elsewhere? If they accidentally ran `cleanse` before copying the file, will this create big problems?

Reply:
Are people at risk of losing data with cleanse?
I'm looking at the description here https://docs.chef.io/server/ctl_chef_backend/#cleanse and looks like there's a risk there.
This steps are to create new cluster, data should be restored from backup taken.There is nothing to loos as restore step will be performed.
Is chef-backend.rb file backed up elsewhere? If they accidentally ran cleanse before copying the file, will this create big problems?